packages/ckeditor5-paste-from-office/src/filters/list.ts

```diff
@@ -178,9 +179,12 @@ function detectListStyle( listLikeItem: ListLikeElement, stylesString: string )
 	const listStyleRegexp = new RegExp( `@list l${ listLikeItem.id }:level${ listLikeItem.indent }\\s*({[^}]*)`, 'gi' );
 	const listStyleTypeRegex = /mso-level-number-format:([^;]{0,100});/gi;
 	const listStartIndexRegex = /mso-level-start-at:\s{0,100}([0-9]{0,10})\s{0,100};/gi;
-	const legalListTypeRegex = new RegExp( `@list l${ listLikeItem.id }:level\\d\\s*{[^{]*mso-level-text:"%\\d\\\\.`, 'gi' );
+	const legalStyleListRegex = new RegExp( `@list l${ listLikeItem.id }:level\\d\\s*{[^{]*mso-level-text:"%\\d\\\\.`, 'gi' );
+	const multiLevelNumberFormatTypeRegex = new RegExp( `@list l${ listLikeItem.id }:level\\d\\s*{[^{]*mso-level-number-format:`, 'gi' );
 
-	const legalListMatch = legalListTypeRegex.exec( stylesString );
+	const legalStyleListMatch = legalStyleListRegex.exec( stylesString );
+	const multiLevelNumberFormatMatch = multiLevelNumberFormatTypeRegex.exec( stylesString );
+	const islegalStyleList = legalStyleListMatch && !multiLevelNumberFormatMatch;
```

Comment:
Maybe we could also avoid checking this if the Multi-level list plugin is not loaded?

Reply:
I think it should not cause performance issue, and also if multi level list plugin is not loaded it doesn't change the fact legal list was detected. It just give us information legal list was found and thats it. Finally our logic will check does multi level list plugin is loaded or not, so checking this 2 times to my mind is not so necessary.